test/fixtures/api/prod-coordinates.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
Where is this file used? Is this a locally stored version of test data to use?

Reply:
We probably don't need this file.  I'll remove it.